pkg/util/util.go

```diff
@@ -33,3 +36,65 @@ func GetRequestHost(req *http.Request) string {
 	}
 	return host
 }
+
+// RefreshValue contains a composite cookie refresh value setting
+type RefreshValue struct {
+	RefreshDuration *time.Duration
```

Comment:
Do these need to be public?

Reply:
Depends. If they are not public, some of your suggestions above can't work and printing logs becomes problematic. I would say it makes it easier if they are publicly accessed without accessor methods.

Reply:
I'm leaving them public as they can be referenced in logging.